ui/ssb-msg.js

```diff
@@ -171,7 +171,96 @@ Vue.component('ssb-msg', {
     else
       self.name = SSB.getProfileName(this.msg.value.author)
 
-    self.body = md.markdown(this.msg.value.content.text)
+    switch(this.msg.value.content.type) {
+      case "about": {
+        var newInfo = "<p>" + this.$root.$t((this.msg.value.author != this.msg.value.content.about ? 'common.profileUpdateOther' : 'common.profileUpdateSelf'), { user: (self.name || this.$root.$t('common.genericUsername')) }) + "</p>"
+        self.body = newInfo
+
+        // Try to find the next-to-latest profile update so we can show what changed.
+        SSB.db.query(
+          and(author(this.msg.value.content.about), type('about')),
```

Comment:
This is not entirely correct as you might have other peoples about messages in here. Meaning author != content.about. Since we only show self-assigned abouts. Meaning we only show that kind of about changes.

Reply:
That was't really understandable ;) Since we only show show self-assigned abouts, I don't think we should display other kind of about changes as it's just confusing.

Reply:
How about that?